### ISSUE TITLE
Cannot read property 'top' of undefined

### ISSUE THREAD

**Issue by ex1st** [snapappointments/bootstrap-select#2163]
Version of bootstrap "3.4.0"
Version of bootstrap-select "1.13.5"

Error throws if a container does not exist (maybe deleted from DOM).
```javascript
/* can be changed to 
if (that.options.container && $container.length && !$container.is('body')) {
*/
if (that.options.container && !$container.is('body')) {
    containerPos = $container.offset();
    containerPos.top += parseInt($container.css('borderTopWidth')); // <--- this place
    containerPos.left += parseInt($container.css('borderLeftWidth'));
} else {
      containerPos = { top: 0, left: 0 };
}
```

**Comment by ex1st** [comment]
Hi @caseyjhol, I've prepared example - https://codepen.io/ex1st/pen/VqyGdo
Thank you.

**Comment by caseyjhol** [comment]
Released in [v1.13.6](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.6)!
